src/compiler/compile_rust_code.ts

```diff
@@ -1,6 +1,19 @@
 import { execSync, IOType } from 'child_process';
 
 export function compileRustCode(
+    dockerContainerName: string,
```

Comment:
It's a little confusing to have this be a required parameter when it is only used when `nativeCompilation === false`

Reply:
Sure, but we always have the dockerContainerName...wouldn't trying to add this in make things maybe unnecessarily complicated?